Fix placeholders (#1352)
redpanda-data/documentation: modules/deploy/pages/redpanda/manual/production/production-readiness.adoc

= Production Readiness Checklist 
:page-aliases: deploy:deployment-option/self-hosted/manual/production/production-readiness.adoc

Before running a production workload on Redpanda, follow this readiness checklist. Redpanda Data recommends using the xref:deploy:redpanda/manual/production/production-deployment-automation.adoc[automated deployment instructions] with Ansible. If you cannot deploy with Ansible, use the xref:deploy:redpanda/manual/production/production-deployment.adoc[manual deployment instructions].

NOTE: For Kubernetes deployments, see the xref:deploy:redpanda/kubernetes/k-production-readiness.adoc[Production Readiness Checklist for Kubernetes].

== Critical requirements

> The critical requirements checklist helps you to confirm that:

Reviewer: ```suggestion
The Critical requirements checklist helps you to confirm that:
```

See also: xref:manage:cluster-maintenance/decommission-brokers.adoc[Decommission Brokers]

== Recommended requirements

> The recommended requirements checklist confirms that you can monitor and support your environment on a sustained basis. It includes the following checks:

Reviewer: ```suggestion
The Recommended requirements checklist confirms that you can monitor and support your environment on a sustained basis. It includes the following checks:
```

* xref:deploy:redpanda/manual/high-availability.adoc#multi-az-deployments[Multi-AZ deployments]
* xref:manage:kubernetes/k-rack-awareness.adoc#configure-rack-awareness[Configure rack awareness in Kubernetes]

== Advanced requirements

> The advanced requirements checklist ensure full enterprise readiness. This indicates that your system is operating at the highest level of availability and can prevent or recover from the most serious incidents. The advanced requirements confirm the following:

Reviewer: ```suggestion
The Advanced requirements checklist ensures full enterprise readiness, indicates that your system is operating at the highest level of availability, and can prevent or recover from the most serious incidents. The advanced requirements confirm the following:
```
